### PR TITLE
arch linux package now in extra

### PR DESCRIPTION
**Pull request by feinedsquirrel**
The Arch Linux "Extra" repo now has `jellyfin-server` and `jellyfin-web`.
The [Jellyfin wiki page](https://wiki.archlinux.org/title/Jellyfin) has been updated.

### CODE REVIEW

**Comment by thornbill** [comment]
It seems the extras package is **not** including Jellyfin's custom ffmpeg build... which is not great 😕

**Comment by feinedsquirrel** [comment]
I am not the maintainer, nor an Arch Linux trusted user. I noticed that it had moved to the official repos, and thought I'd get the ball rolling.

Something that may be possible is to have the package maintainer add an `*.install` script informing of the existence of the `jellyfin-ffmpegX` packages in the AUR. Looking now, it seems strange that `jellyfin-ffmpeg` is only v4, and not "current", while `jellyfin-ffmpeg6` appears to be the latest version. Arch convention seems to be the `jellyfin-ffmpeg` would always be "latest", and older versions get their version number attached. Discussion with a Trusted User might clarify whether this is a recommended change. @nyanmisaka should probably be involved in that discussion.

Last thought I had was wondering if the jellyfin-ffmpeg alterations get merged upstream. From some reddit comments, and looking myself, jellyfin's version is generally kept close to upstream, and just gets a few patches applied on top. If those patches regularly get merged upstream, then it shouldn't matter that the Arch package links to the system `ffmpeg`, since it stays with upstream stable reliably (in my experience).

At a minimum, I can remove the old content in this documentation, and replace it with a note that Arch users may consider installing `jellyfin-ffmpegX` from the AUR.

Finally, I have not looked at the download wizard, but from the wording of @enter-a-random-username 's comment, it sounds like there is more there than simply altering something related to Arch. Should it be a separate pull-request? (and thinking about it more, it may not even be in this repo, not sure.)

**Comment by nyanmisaka** [comment]
Many changes in jellyfin-ffmpeg are not suitable for upstream, or would take forever, so we decided to maintain a fork to meet our needs. The existence of `jellyfin-ffmpeg{5,6}` was due to changes in the upstream ffmpeg API that broke compatibility with jellyfin-server. At that time, I never considered that jellyfin-server AUR would be adopted by extra.

Maybe I should get in touch with their maintainer to add jellyfin-ffmpeg into the jellyfin-server PKGBUILD, like what emby-server PKGBUILD does. But I didn't find the an email or other contact information.

https://gitlab.archlinux.org/archlinux/packaging/packages/jellyfin-server/-/blob/main/PKGBUILD
https://gitlab.archlinux.org/archlinux/packaging/packages/emby-server/-/blob/main/PKGBUILD

**Comment by nyanmisaka** [comment]
https://github.com/grawlinson

Well I found it, thanks to him using the same username on github.
I'll try to get in touch with him and find a good place for jellyfin-ffmpeg.

**Comment by feinedsquirrel** [comment]
Not that you need my approval, but putting `jellyfin-ffmpeg` into the `*-server` pkgbuild seems like a good idea. And thank you for clarifying the differences of that fork. The reddit comments I had found were not super detailed.

I held off for now making a recommendation to install jellyfin-ffmpeg from the aur alongside the Arch "Extra" version until we hear the outcome of the discussion between @nyanmisaka and @grawlinson. If the decision is made to keep them separate, then we can add a phrase in this documentation, and to the Jellyfin page on the Arch Wiki that a `*-server` installation may be well served by also installing the package from the AUR.

**Comment by grawlinson** [comment]
I see someone has pinged me! Can we move this to a separate issue (as it's not relevant to this PR) and discuss it there?

**Comment by feinedsquirrel** [comment]
@nyanmisaka might have input as well on where a separate issue should be opened; he's also messaged you directly, I believe. I wonder if an arch flyspray issue would be best, since it primarily involves how it is packaged in the arch repo.

**Comment by nyanmisaka** [comment]
> I see someone has pinged me! Can we move this to a separate issue (as it's not relevant to this PR) and discuss it there?

Let's move here. https://github.com/jellyfin/jellyfin/issues/10315

**Comment by feinedsquirrel** [comment]
@enter-a-random-username I've taken a stab at the download wizard.

I looked through the other options in the file as a guide.
I wonder if instead of linking to the package in Arch-Extra, it should  link to the Arch Wiki page.
Anyone have thoughts on this?

**Review by Shadowghost — APPROVED** [review]

**Comment by feinedsquirrel** [comment]
`jellyfin-bin` has now been removed from the AUR, so I removed it from the doc. I also explicitly stated `-server-git` and `-web-git` are also available.

I'm not sure if anything needs to be mentioned about older versions of `jellyfin-ffmpeg` in the AUR. Let me know if I should add anything. Otherwise, I think this is about ready, as the linked issues have been resolved.

**Pull request merged**
